ipynb/examples/android/workloads/Android_Viewer.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "# Generic Android viewer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "from conf import LisaLogging\n",
+    "LisaLogging.setup()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "%pylab inline\n",
+    "\n",
+    "import json\n",
+    "import os\n",
+    "\n",
+    "# Support to access the remote target\n",
+    "import devlib\n",
+    "from env import TestEnv\n",
+    "\n",
+    "# Import support for Android devices\n",
+    "from android import Screen, Workload, System\n",
+    "from target_script import TargetScript\n",
+    "from viewer import Viewer\n",
```

Comment:
Maybe we can avoid this by moving the definition of the `Viewer` class beside that of `Workload`?
That way it should also be possible to export `Viewer` (which I would also rename into `ViewerWorkload`) from the same `__init__.py` which already exports `Worklaod`, i.e. `libs/utils/android/__init__.py`.

In that case here it should be possible to have just:
```python
from android import Screen, System, ViewerWorkload
```

Not sure will work... but perhaps worth to try ;-)